### ISSUE TITLE
Enable RSS autodiscovery

### ISSUE THREAD

**Issue by Sancus** [thundernest/thunderblog#12]
Think we need something like `<link rel="alternate" type="application/rss+xml" title="Thunderbird Blog" href="<?php bloginfo(‘rss_url’); ?>" />` in the header.php

**Comment by freaktechnik** [comment]
Alternatively there's also https://codex.wordpress.org/Automatic_Feed_Links if you use `wp_head()` I think. To get the actual URL there's also the specialized https://developer.wordpress.org/reference/functions/get_feed_link/

**Comment by devmount** [comment]
Yes, I would go with @freaktechnik's approach. Since we already use `wp_head()` adding

```php
add_theme_support( 'automatic-feed-links' );
```

would automatically add the posts feed and the comments feed to the header:

```html
<link rel="alternate" type="application/rss+xml" title="Thunderbird Blog &raquo; Feed" href="https://blog.thunderbird.net/feed/" />
<link rel="alternate" type="application/rss+xml" title="Thunderbird Blog &raquo; Comments Feed" href="https://blog.thunderbird.net/comments/feed/" />
```
